src/web/theme/ui/templates/Page/Page.scss

```diff
@@ -1,5 +1,26 @@
 .page {
-  @include horizontal-spacing();
   max-width: $desktop;
-  margin: 3rem auto;
+  display: flex;
+  justify-content: center;
+  position: relative;
+  overflow: hidden;
+  padding: 0 calc((100vw - #{$tablet}));
+  h1 {
+    text-shadow: 0 2px 4px rgba(0, 0, 0, 0.35);
+    color: $white;
+  }
+}
+
+@media screen and (min-width: $desktop) {
+  .page {
+    padding: 0 calc((100vw - #{$desktop}));
+  }
+}
+
+@media screen and (min-width: $largeDesktop) {
+  .page {
+    //Here I had to use 75rem due to tests for padding weird comportement
```

Comment:
I don't think the behavior of the padding itself is weird, nor that this comment is very useful. You did tests to find the value that suited your needs and you found it, that's all.

Reply:
ok :cry: